test/test_ecs_monitor.py

```diff
@@ -330,22 +330,17 @@ def describe_services_generator(final_running_count):
 
         event_list = [e for e in events]
 
-        assert len(event_list) == 3
+        assert len(event_list) == 7
```

Comment:
We should be able to update the fixtures to account for the new logic, leave the assertions as they were and keep this test testing what it was originally testing.